File: 1030/Hw24.c
#include <stdio.h>
#include <stdlib.h>

int main() {
	char fiWord[10];
	char seWord[10];
	int fiCount;
	int seCount;
	char compare;
	
	printf("Enter the word : ");
	fgets(fiWord, sizeof(fiWord), stdin);
	printf("Enter the word : ");
	fgets(seWord, sizeof(seWord), stdin);
	for(fiCount = 0; fiCount <strlen(fiWord);fiCount++)
		for (seCount = 0; seCount < strlen(fiWord); seCount++) {
			if (seWord[fiCount] > seWord[seCount])
			{
				compare = seWord[fiCount];
				seWord[fiCount] = seWord[seCount];
				seWord[seCount] = compare;
			}
			if (fiWord[fiCount] > fiWord[seCount]) {
				compare = fiWord[fiCount];
				fiWord[fiCount] = fiWord[seCount];
				fiWord[seCount] = compare;
			}
		}
	if (!strcmp(fiWord, seWord)) {
		printf("Two words are anagram");
	}
	else
		printf("Two words are not anagram");

	system("pause");
}

File: 1030/hw24.java
import java.util.Scanner;

public class hw24 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        System.out.print("Enter the first word : ");
        String word = scan.next();
        System.out.print("Enter the second word: ");
        String word2 = scan.next();
       char fiResult[] = word.toCharArray();
       char seResult[] = word2.toCharArray();
        if (word.length() == word2.length()) {
            for(int b = 0; b < word.length();b++)
            for (int a = 0; a < word.length(); a++){
                char compare;
                if(fiResult[b] < fiResult[a]){
                    compare = fiResult[b];
                    fiResult[b] = fiResult[a];
                    fiResult[a] = compare;
                }
                if(seResult[b] < seResult[a]){
                    compare = seResult[b];
                    seResult[b]= seResult[a];
                    seResult[a] = compare;
                }
            }
            word = new String(fiResult,0,fiResult.length);
            word2 = new String(seResult,0,seResult.length);

                if (word.equals(word2)) {
                    System.out.print("Two words are same");
                } else
                    System.out.print("Two words are different");

        }
    }
}
